transfer-data/destinations.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: 'Destinations'
+description: 'This page explains how to set up destinations.'
+---
+
+<Note>
+Flow is currently in preview. To try it out, [sign up for a free preview](https://forms.axiom.co/flow).
+</Note>
+
+To transfer and transform data from an Axiom dataset to a destination, you need to set up a destination. This is where data is transferred. Once you set up a destination, you can use it in any flow.
+
+To set up a destination:
+1. Click the **Flows** tab. Axiom displays the list of flows you have created.
+2. In the top right, click **Manage destinations**, and then click **New destination**.
+3. Name the destination.
+4. In **Kind**, select the destination type.
+5. Configure the destination. For more information on each destination type, see the following:
+
+    - [AWS S3](#configure-aws-s3-destination)
+    - [Axiom](#configure-axiom-dataset-destination)
+    - [Azure](#configure-azure-destination)
+    - [Elastic Bulk](#configure-elastic-bulk-destination)
+    - [HTTP](#configure-http-destination)
+    - [HTTP (Authorization)](#configure-http-authorization-destination)
+    - [HTTP (Basic)](#configure-http-basic-destination)
+    - [OpenTelemetry Traces](#configure-opentelemetry-traces-destination)
+
+6. In the bottom right, click **Save**.
+
+## Configure AWS S3 destination
```

Comment:
Great 👍  Is navigation easier if these are each dedicated pages?

Reply:
I am not sure how to approach this. I wonder if we should, or not, give some notion in terms of destination implementation maturity? For instance, Axiom destinations are still a bit problematic, and Elastic Bulk hasn't been tested with big loads yet.

Reply:
I agree with separate pages for each destination. Better for visibility, SEO, and scalability. If we want to ship this today, I would break this page up in another PR.